### PR TITLE
Project admins can create networkpolicy objects

### PR DESCRIPTION
**Pull request by bfallonf**
From https://trello.com/c/YAWd5u0R/584-document-roles-policies-for-networkpolicy-policy

@danwinship Any thoughts?

### CODE REVIEW

File: admin_guide/managing_networking.adoc [hunk 1]
```diff
@@ -1081,6 +1081,9 @@ other projects. To isolate a project, opt in to isolation by configuring the
 Namespace object, then create a `NetworkPolicy` object to indicate the allowed
 incoming connections.
 
+Project administrators can create and delete NetworkPolicy objects within their
+own project.
```
Comment (danwinship) [inline admin_guide/managing_networking.adoc]: Text is good but maybe move it further down, to the "Create the `NetworkPolicy` objects" item?

**Comment by bfallonf** [comment]
Thanks @danwinship . Changed to suggestion.

@gaurav-nelson PTAL.

**Comment by gaurav-nelson** [comment]
@bfallonf All good :mega:

**Comment by bfallonf** [comment]
Thanks @gaurav-nelson . I'll merge.

**Pull request merged**